### PR TITLE
[aml] fix realtime streams after clock interface removal

### PR DESCRIPTION
**Pull request by stefansaraev**
this fixes live tv after #9152

@FernetMenta @fritsch @koying

### CODE REVIEW

**Comment by fritsch** [comment]
Thank you for looking into this. If you like you can also fix the usage of m_player_pts which just records the player pts timestamps and might have no relation with the pic we return in GetPicture().

Btw. the other thing still missing is: we have no possibility to drop in the decoder as of now, so a/v sync is still a dice game.

**Comment by fritsch** [comment]
jenkins build this please

**Comment by stefansaraev** [comment]
this still has issues with some hd channels (most work here, sd always work, local content also work), I'll see if increasing buffer usage can help.

> Btw. the other thing still missing is: we have no possibility to drop in the decoder as of now, so a/v sync is still a dice game.

not easy for someone like me, who doesnt understand most of the code ;)

File: xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp [hunk 1]
```diff
@@ -1677,6 +1676,10 @@ void CAMLCodec::CloseDecoder()
   SysfsUtils::SetInt("/sys/class/tsync/enable", 1);
 
   ShowMainVideo(false);
+
+  // add a little delay after closing in case
+  // we are reopened too fast.
+  usleep(500 * 1000);
```
Comment (Jalle19) [inline xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp]: Wasn't the root cause for this found already?

Comment (stefansaraev) [inline xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp]: not really

**Comment by stefansaraev** [comment]
na. more needs to be fixed, this is useless. I'll just close it for now. sorry for the noise.

**Pull request closed without merging**